.github/workflows/check-links.yaml

```diff
@@ -0,0 +1,27 @@
+name: Check Links
+
+on:
+  repository_dispatch:
+  workflow_dispatch:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  linkChecker:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Link Checker
+        id: lychee
+        uses: lycheeverse/lychee-action@v1.5.0
+        env:
+          GITHUB_TOKEN: ${{secrets.GITHUB_TOKEN}}
+
+      # - name: Create Issue From File
```

Comment:
Is this here by mistake?

Reply:
No, it's commented out to remind me to revisit this later. The maintainer recently fixed a bug I filed regarding auto-opening of issues found by this action, I just haven't had time to verify.